firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -83,4 +86,15 @@ private static String encodeFidBase64UrlSafe(byte[] rawValue) {
     bb.putLong(uuid.getLeastSignificantBits());
     return bb.array();
   }
+
+  public static long getCurrentTimeInSeconds() {
+    return System.currentTimeMillis() / 1000;
+  }
+
+  public static boolean isNetworkAvailable(Context context) {
```

Comment:
as above, I'd inject a "NetworkChecker" or similar so that you're not forced to muck around with a real ConnectivityManager in tests